tests/runfiles/linux.run

```diff
@@ -801,3 +801,9 @@ tags = ['functional', 'zvol', 'zvol_swap']
 [tests/functional/libzfs]
 tests = ['many_fds']
 tags = ['functional', 'libzfs']
+
+[tests/functional/pyzfs]
+tests = ['pyzfs_unittest']
+pre =
+post =
+tags = ['functional', 'libzfs']
```

Comment:
Did you mean `pyzfs` here instead of `libzfs`?   Let's also keep this in alphabetical order.

Reply:
I was considering pyzfs as being part of the "libzfs(core)" tests, hence the "libzfs" tag: will fix in the next push.